### PR TITLE
Implement Trinkets integration

### LINKED ISSUE #39: Fabric: Add support for Elyra Slot/Trinket
**Feature for problem**
There are currently a couple interchangeable mods that allow users to wear an Elyta at the same time as a chest armor, via the [Trinkets](https://modrinth.com/mod/trinkets) API: [Elytra Trinket](https://modrinth.com/mod/elytra_trinket) and [Elytra Slot](https://modrinth.com/mod/elytra-slot).

It would be awesome if, Simple Armor Hud could detect these mods and display an additional item in the HUD for the Elytra when it's in the "cape trinket slot"!

Thanks!

### PR DESCRIPTION
**Pull request by cerus**
Implements the [Trinkets](https://github.com/emilyploszaj/trinkets) api. Resolves #39 

image

### CODE REVIEW

**Comment by legoraft** [comment]
This PR also moves logging to a dedicated logger. That's great, this is mostly for my own information. Did you also try and mess around with the `renderArmorPiece` function to make the elytra appear less close to the chestplate?

File: build.gradle [hunk 1]
```diff
@@ -19,6 +19,14 @@ repositories {
 	// for more information about repositories.
 	maven { url = "https://maven.terraformersmc.com/" }
```
Comment (legoraft) [inline build.gradle:20]: This line should be removed, as the maven has been re-added from line 22 and on

File: src/client/java/com/armorhud/armorHud.java [hunk 2]
```diff
@@ -1,20 +1,34 @@
...
+        LOGGER.info("Simple Armor Hud loaded!");
         CONFIG.load();
         armorHudRegistries.registerArmorHud();
         handleKeys();
+
+        armorAccessor = new VanillaArmorAccessor();
+        if (FabricLoader.getInstance().isModLoaded("trinkets")) {
+            armorAccessor = new TrinketsArmorAccessor(armorAccessor);
+        }
+        LOGGER.info("Armor accessor implementation: {}", armorAccessor.getClass().getSimpleName());
```
Comment (legoraft) [inline src/client/java/com/armorhud/armorHud.java:31]: The `util/modDetect` file has been made to detect if mods are loaded, could this be moved there?

**Comment by cerus** [comment]
> Did you also try and mess around with the renderArmorPiece function to make the elytra appear less close to the chestplate?
No, I didn't touch that function.

**Comment by legoraft** [comment]
> > Did you also try and mess around with the renderArmorPiece function to make the elytra appear less close to the chestplate?
> 
> No, I didn't touch that function.

I'll take a look at it at another time, it's great that everything's functional

**Comment by legoraft** [comment]
Thanks for your help, I'll try and draft a release asap

**Pull request merged**